packages/abstract-app/src/interface.rs

```diff
@@ -0,0 +1,100 @@
+#[macro_export]
+/// Creates the interface for working with the app wth cw-orch
+macro_rules! create_interface {
```

Comment:
can you apply this in the example as well?

Reply:
https://github.com/AbstractSDK/contracts/pull/371/commits/60e6ec116288334c5c343c5721a89408cf973d77